examples/gaussian_processes/gaussian_process.ipynb

```diff
@@ -4,18 +4,23 @@
    "cell_type": "markdown",
```

Comment:
<p>The readthedocs error is due to the post directive. The date needs to be present and in the format shown in the jupyter style guide: <code>mmm DD, YYY</code>.The actual day doesn't really matter, as long as it is relatively close to the last update date, it can be today: <code>Jul 31, 2022</code>  , but could as well be the day the PR was opened or some day next week.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated